commands/info.js

```diff
@@ -0,0 +1,172 @@
+	return new Promise((resolve, reject) => {
+		try {
+			if (cliInfo) {
+				// Display info on CLI itself
+				const gm = globalModules();
+				const gCLI = path.join(gm, '@enact', 'cli');
+				const isGlobal =
+					fs.existsSync(gCLI) &&
+					path.dirname(require.resolve(path.join(gCLI, 'package.json'))) === path.dirname(__dirname);
+				console.log(chalk.yellow.bold('==Enact CLI Info=='));
+				if (isGlobal && fs.lstatSync(gCLI).isSymbolicLink()) {
+					const ver = gitInfo(__dirname) || require('../package.json').version;
+					console.log(`Enact CLI: ${ver}`);
+					console.log(chalk.cyan('\tSymlinked from'), path.dirname(__dirname));
+				} else {
+					console.log(`@enact/cli: ${require('../package.json').version}`);
+				}
+				console.log(`Installed Globally: ${isGlobal}`);
+				if (isGlobal) console.log(`Global Modules: ${gm}`);
+				console.log();
+
+				// Display info on in-house components, likely to be linked in
+				console.log(chalk.yellow.bold('==Enact Components=='));
+				[
+					'@enact/dev-utils',
+					'eslint-config-enact',
+					'eslint-plugin-enact',
+					'postcss-resolution-independence'
+				].forEach(dep => logVersion(dep));
+				console.log();
+
+				// Display info on notable 3rd party components
+				console.log(chalk.yellow.bold('==Third Party Components=='));
+				console.log(`Babel: ${require('@babel/core/package.json').version}`);
+				console.log(`ESLint: ${require('eslint/package.json').version}`);
+				console.log(`Jest: ${require('jest/package.json').version}`);
+				console.log(`LESS: ${require('less/package.json').version}`);
+				console.log(`Webpack: ${require('webpack/package.json').version}`);
+			} else {
+				const app = require('@enact/dev-utils').optionParser;
+				const meta = require(path.join(app.context, 'package.json'));
+				const bl = require(resolveSync('browserslist', {
+					basedir: path.dirname(require.resolve('@enact/dev-utils/package.json')),
+					preserveSymlinks: false
+				}));
+				app.setEnactTargetsAsDefault();
+				console.log(chalk.yellow.bold('==Project Info=='));
+				console.log(`Name: ${app.name}`);
+				console.log(`Version: ${gitInfo(app.context) || meta.version}`);
+				console.log(`Path: ${app.context}`);
+				console.log(`Theme: ${(app.theme || {}).name}`);
+				if (app.proxer) console.log(`Serve Proxy: ${app.proxy}`);
+				if (app.template) console.log(`Template: ${app.template}`);
+				if (app.externalStartup) console.log(`External Startup: ${app.externalStartup}`);
+				if (app.deep) console.log(`Deep: ${app.deep}`);
+				if (app.forceCSSModules) console.log(`Force CSS Modules: ${app.forceCSSModules}`);
+				console.log(`Resolution Independence: ${JSON.stringify(app.ri)}`);
+				console.log(`Browserslist: ${bl.loadConfig({path: app.context})}`);
+				console.log(`Environment: ${app.environment}`);
+				console.log();
+				console.log(chalk.yellow.bold('==Dependencies=='));
+				Object.keys(meta.dependencies).forEach(dep => {
```

Comment:
This can throw if the `package.json` has no `.dependencies`.  Though rare, I actually had a directory with this, so I added a conditional here.

Reply:
Each fix, will update, thanks for spotting